### ISSUE TITLE
How to assign a user to a pull request recently created?

### ISSUE THREAD

**Issue by evandrocoan** [PyGithub/PyGithub#1009]
![image](https://user-images.githubusercontent.com/5332158/50737362-54bafb80-11af-11e9-8dad-74ab04d8dcae.png)

```python
fork_user = github_api.get_user('user')
fork_repo = fork_user.get_repo('repo')

fork_pullrequest = fork_repo.create_pull("Title", "Body", 'master', 
          '{}:{}'.format('otheruser', 'master'), False)
```

After running this code, I got the pull_request object. Looking over its documentation at:
1. https://pygithub.readthedocs.io/en/latest/github_objects/PullRequest.html

I could not find a method to assign a user to the pull request like:
```python
fork_pullrequest.assign(fork_user)
```

**Comment by rallytime** [comment]
@evandrocoan I think what you might be looking for is the `create_review_request` function [here](https://pygithub.readthedocs.io/en/latest/github_objects/PullRequest.html#github.PullRequest.PullRequest.create_review_request).

**Comment by rallytime** [comment]
Hi @evandrocoan - I implemented this feature request in #1016.
